sql-cli/src/sql_cli/main.py

```diff
@@ -0,0 +1,53 @@
+from argparse import ArgumentParser, Namespace
+from datetime import datetime
+from pathlib import Path
+from typing import Tuple
+
+from sql_cli.dag_generator import SqlFilesDAG
+from sql_cli.sql_directory_parser import get_sql_files
+from sql_cli.utils import render_jinja
+
+
+def cli() -> Namespace:
+    parser = ArgumentParser()
+    parser.add_argument("directory", type=Path)
+    args = parser.parse_args()
+    return args
```

Comment:
This is temporary until we switch to click?

Reply:
This is correct.